cleanlab_studio/internal/api/api.py

```diff
@@ -379,3 +380,58 @@ def get_prediction_status(api_key: str, query_id: str) -> Dict[str, str]:
     handle_api_error(res)
 
     return cast(Dict[str, str], res.json())
+
+
+def tlm_prompt(
+    api_key: str,
+    prompt: str,
+    quality_preset: str,
+) -> JSONDict:
+    """
+    Prompt Trustworthy LLM with a question, and get back its answer along with a confidence score
+
+    :param api_key:
```

Comment:
Why two different docstring formats? (compare with TLM.prompt, for example)

Reply:
was just keeping what Kat had written. can convert both to google docstrings